pkg/cmd/grafana-cli/commands/commands.go

```diff
@@ -30,7 +29,7 @@ func runDbCommand(command func(commandLine utils.CommandLine, sqlStore *sqlstore
 			return errutil.Wrap("failed to load configuration", err)
 		}
 
-		if debug {
+		if context.IsSet("debug") {
```

Comment:
This should be wrong AFAICT. `IsSet` tells you if the flag is set, not the value of it is. Besides, the old code for getting the flag value is fine.

Reply:
I am not sure I understand. A boolean flag does not have (or at least does nto need) a value? It's presence determines either debug mode is set, defaulting to no. Isn't that logic right?

Reply:
Well, reading https://godoc.org/github.com/urfave/cli again... Things are foggy to me at best. Seems like the `IsSet` & `Bool` functions are somewhat colliding and/or redundant...